ui/raidboss/data/05-shb/dungeon/akadaemia_anyder.txt

```diff
@@ -61,6 +61,7 @@ hideall "--sync--"
 # -p 3E16:508.5 -ii 3E11 3E14
 500.0 "--sync--" sync /00:0839:Phytobiology will be sealed off/ window 500,0
 505.0 "--sync--" sync /:Marquis Morbol:3E14:/ window 505,5
+508.0 "--sync--" sync /:Marquis Morbol:3E16:/ window 508,5
```

Comment:
I'm not sure what you're trying to do.  It looks like you're adding an additional sync for lash (`3E16`?).  Which Arbor Storm is being skipped? The one at 524.7?

This is old, but is the best explanation for timelines: https://dtguilds.enjin.com/mobile/forum/viewthread/m/37032836/id/26353492-act-timeline-plugin